account-kit/signer/src/client/alchemy.ts

```diff
@@ -0,0 +1,91 @@
+import type { SignerBody, SignerResponse, SignerRoutes } from "./types.js";
+import { VERSION } from "../version.js";
+import type { ConnectionConfig } from "@aa-sdk/core";
+import type { TSignedRequest } from "@turnkey/http";
+
+export class AlchemySignerClient {
```

Comment:
This should implement all the other routes as well if it's going to exist (signer config, prepare oauth, etc) though I'm not super clear on why the base signer client no longer impls this stuff since that's the base functionality imo (I'm probably missing some context)

Reply:
this will be removed in upcoming commit